onnxscript/values.py

```diff
@@ -195,6 +195,16 @@ def param_schemas_from_op_schema(
     return tuple(schemas)
 
 
+def _is_optional(typeinfo) -> bool:
+    if typing.get_origin(typeinfo) is Union and type(None) in typing.get_args(typeinfo):
+        # Python < 3.10
+        return True
+    if typing.get_origin(typeinfo) is Optional:
+        # Python >= 3.10
```

Comment:
I think there’s a similar function in onnx types or somewhere else. Let me double check

Reply:
Maybe update https://github.com/microsoft/onnxscript/blob/8521b27c9a549614c7d5250e508ece23c27e3297/onnxscript/type_annotation.py#L141?

Reply:
thanks for pointer